nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
It is not guaranteed to be correct. According to the standard, `void *` is a data pointer, not a function pointer. Function pointers may be of different size or refer to other memory areas than data pointers.

https://stackoverflow.com/questions/12358843/why-are-function-pointers-and-data-pointers-incompatible-in-c-c